Gaijinizer patch/Map216.txt

```diff
@@ -454,30 +474,30 @@ stopped". It doesn't really stop time. .#MTLed
 そもそも『時間停止』なぞ、規模が尋常ではない。
 その世界の全ての次元が範囲じゃぞ？
 > CONTEXT: Map216/events/5/pages/1/168/Dialogue/enemy17
-Rather...that twins was out of spec.
-In the first place, the scale is not unusual as
-it is "time stop". All the dimensions of the
-world are in range?#MTLed
+But the Twins are quite extraordinary.
+Stopping time is a rather uncommon ability, and
+that ability includes all of this world's
+dimensions.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 同じ世界の時空を自由に行き来できる悪魔……
 
 だからこそ、あんな芸当ができるのであろう。
 > CONTEXT: Map216/events/5/pages/1/173/Dialogue/enemy17
-You can freely travel around the same world time
-space...demon ...... That's why you can do that
-kind of trick.#MTLed
+They're a demon that can freely move through
+space and time in the same world, which is why
+they have that kind of amazing ability...#RonIffyWording
 > END STRING
```

Comment:
Follow up from previous:
>_**They're a demon that can freely move around
a world's spacetime at will... That's probably why
they have that kind of amazing ability.**_